layouts/shortcodes/button.html

```diff
@@ -1,7 +1,8 @@
 {{ $base_url := .Site.BaseURL }}
+{{ $lang := .Site.Language.Lang }}
 <a
   class="rounded-md bg-primary-600 px-4 py-2 text-neutral no-underline hover:bg-primary-500 hover:rounded-md"
-  {{ with .Get "href" }}href="{{ $base_url }}{{ . }}"{{ end }}
+  {{ with .Get "href" }}href="{{ $base_url }}{{ $lang }}{{ . | relURL }}"{{ end }}
```

Comment:
```suggestion
  {{ with .Get "href" }}href="{{ . | relLangURL }}"{{ end }}
```

I gave this a go with `hugo server`. I had an intuition from localising other sites that there must be some sort of function to generate a URL with a language code: https://gohugo.io/content-management/multilingual/#multilingual-themes-support

The reason for removing the `{{ $base_url }}` is that in theory it's redundant anywho. Also , `$base_url` ends with a `/`, and `rel[Lang]URL` starts with a `/` so it would end up with double slashes.

Thoughts?

Reply:
Yes, sounds great, thanks for identifying!